src/server/tiered_storage_test.cc

```diff
@@ -47,6 +47,15 @@ class TieredStorageTest : public BaseFamilyTest {
 
     BaseFamilyTest::SetUp();
   }
+
+  void TearDown() override {
+    TieredStats stats;
+    do {
+      util::ThisFiber::SleepFor(20ms);
```

Comment:
fine but what about production use-case? if we shutdown with io requests in flight we may get random segfaults